constants.sh

```diff
@@ -128,6 +128,25 @@ if [[ "${TESTNET}" == "local" ]]; then
   then
     export REGISTRY_PATH="$(readlink -f "${REGISTRY}")"
   fi
```

Comment:
I'd remove the original candidate paths as they are certainly not gonna work without `find "$BASE_PATH" -type d -name ic_registry_local_store`.

Reply:
Done!